stage #0/module #4 Algorithms, Data Structures/README.md

```diff
@@ -0,0 +1,28 @@
+<h1 align="center">Algorithms. Data structures</h1>
```

Comment:
Before creating this PR, it would be nice to merge with the following PR https://github.com/sjubusel/MJC-School/pull/1 . In order to understand, what materials were added.